server/config.go

```diff
@@ -308,7 +308,7 @@ type ScheduleConfig struct {
 	// ReplicaScheduleLimit is the max coexist replica schedules.
 	ReplicaScheduleLimit uint64 `toml:"replica-schedule-limit,omitempty" json:"replica-schedule-limit"`
 	// Schedulers support for loding customized schedulers
-	Schedulers SchedulerConfigs `toml:"schedulers,omitempty" json:"schedulers"`
+	Schedulers SchedulerConfigs `toml:"schedulers,omitempty" json:"schedulers-v2"` // json v2 is the for sake of compatible upgrade
```

Comment:
is the for -> is for.

Reply:
"the" is necessary here, it's idiomatic usage

Reply:
"the" is necessary, but it should after "for", I mistake here...

Reply:
got it.

Reply:
why `-v2` here?

Reply:
schedulerConfigs struct in old version is different from new version, so when reload persisted old version json string, new version can not unmarshal correctly. Thus change the name of this field, new version will ignore the old version info which in field "schedulers" and store new info in "schedulers-v2".